dapp/src/components/_AccountStatusContent.js

```diff
@@ -85,7 +87,19 @@ const AccountStatusContent = ({ className, onOpen }) => {
             </>
           )}
         </div>
-        {active && (
+        {overrideAccount && (
+          <div className="disconnect-box d-flex">
+            <a
+              className="btn-clear-blue w-100"
+              onClick={() => {
+                window.location = window.location.pathname
```

Comment:
What's the purpose of using this? To not use the `overrideAccount`? If that's the case, why not use React Router's hooks and functions to do it?

Reply:
this is just to refresh the page without the override_account params. do you think we should use `router.push('/history)` instead?

Reply:
Yeah but `replace` rather than `push`, I'd say that'd be a lot cleaner

Reply:
sure